src/app/migration/src/src/app/modules/workspace/components/content-creation-forms/create-collection/create-collection.component.ts

```diff
@@ -125,18 +132,23 @@ generateData() {
   /***
    *   Create colletion creates the content Id and navigate
    * */
-  createCollection() {
+  createCollection(modalCollection) {
+    this.showLoader = true;
    const requestData = {
       content: this.generateData()
     };
+    this.loaderMessage = {
+      'loaderMessage': this.resourceService.messages.stmsg.m0016,
+  };
     this.editorService.create(requestData).subscribe(res => {
       this.showLoader = false;
-      const type = 'CollectionEditor';
-      const state = '';
+      const type = 'Collection';
+      const state = 'state';
       const framework = 'framework';
```

Comment:
framework api should be called to fetch the value, hardcoding string is not accepted.

Reply:
As per rajath, it is required, he will be taking the data from the api and update it, which he had done in Angular 1.